packages/edit-site/src/components/sidebar/global-styles-v2/components/inspector.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { motion } from 'framer-motion';
+import ReactJson from 'react-json-view';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Card,
+	CardBody,
+	CardHeader,
+	FormGroup,
+	Panel,
+	PanelBody,
+	PanelHeader,
+	Select,
+	useNavigatorHistory,
+	useNavigatorLocation,
+	View,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { useAppState } from '../state';
+
+const routes = [
```

Comment:
Do we need navigation to persist in the URL?

Reply:
@youknowriad do you think it's a useful feature for navigation to persist in the URL? For FSE it might be nice for someone to be able to share a link directly to the global styles pane they were looking at for collaboration, right?

Reply:
Deep linking is likely going to be needed in some form but I'm not sure if we want to take the overhead of a full router. This is more hazy for the other instance of "navigator" on the W menu.